huaweicloud/services/acceptance/sfs/resource_huaweicloud_sfs_turbo_test.go

```diff
@@ -372,7 +372,7 @@ func testAccSFSTurbo_basic(rName string) string {
 	return fmt.Sprintf(`
 %s
 
-data "huaweicloud_availability_zones" "test" {}
+data "huaweicloud_er_availability_zones" "test" {}
```

Comment:
Why do change `huaweicloud_availability_zones` to `huaweicloud_er_availability_zones` ？